MODULE.bazel

```diff
@@ -1019,6 +1028,10 @@ use_repo(
     "policy_pinned_testing",
     "unpinned_policy_pinned_testing",
 
+    # Pinned repo
+    "transitive_dependency_with_type_of_pom",
+    "unpinned_transitive_dependency_with_type_of_pom",
```

Comment:
Nit: we no longer need the `unpinned_` variant since `//transitive_dependency_with_type_of_pom:pin` will work

Reply:
I should go through and remove more, but I guess if we add this it'll be one more line :)